pkg/connector/database_users.go

```diff
@@ -240,31 +282,37 @@ func (o *databaseUserBuilder) CreateAccountCapabilityDetails(ctx context.Context
 	}, nil, nil
 }
 
-func (o *databaseUserBuilder) createUserIfNotExists(ctx context.Context, orgId, email string, profile map[string]any) error {
+func (o *databaseUserBuilder) createUserIfNotExists(ctx context.Context, orgId, email string, profile map[string]any) (string, error) {
 	l := ctxzap.Extract(ctx)
 
-	_, httpResponse, err := o.client.OrganizationsApi.CreateOrganizationInvitation(
+	userInvite, httpResponse, err := o.client.MongoDBCloudUsersApi.CreateOrganizationUser(
 		ctx,
 		orgId,
-		&admin.OrganizationInvitationRequest{
-			Username:             &email,
-			Roles:                parseStrList(profile["roles"], []string{"ORG_MEMBER"}),
-			TeamIds:              parseStrList(profile["teamIds"], []string{}),
-			GroupRoleAssignments: nil,
+		&admin.OrgUserRequest{
+			Username: email,
+			Roles: admin.OrgUserRolesRequest{
+				OrgRoles:             *parseStrList(profile["roles"], []string{"ORG_MEMBER"}),
```

Comment:
can roles be empty? if so, is this safe for empty roles

Reply:
It's safe, `parseStrList` checks the value